Go.gitignore

```diff
@@ -10,3 +10,6 @@
 
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
+
+# Dependency directories
```

Comment:
```suggestion
# Dependency directories (remove the comment below to include it)
```

Reply:
I would rather see ignoring opt-in, not opt-out.